.changeset/five-spoons-matter.md

```diff
@@ -0,0 +1,5 @@
+---
+"@google/generative-ai": minor
+---
+
+Added `responseSchema` to `GenerationConfig` to allow for JSON schema reponses.
```

Comment:
suggest "to allow user to provide a JSON schema when `responseMimeType` is set to JSON."